recipe/README.md

```diff
@@ -1,3 +1,10 @@
+Recipes for Tensorflow related conda packages for tensorflow >1.8.0,<=2.12 can be found at:
+https://github.com/AnacondaRecipes/tensorflow_recipes
+
+We are moving back to using this feedstock as of tensorflow 2.17.0.
+
+
```

Comment:
This isn't a fork of conda-forge's feedstock. Whoever works on this and wishes to bring changes from the conda-forge feedstock should pull conda-forge's changes and reapply changes made in this PR on top of that.